routes/vetoes.js

```diff
@@ -181,26 +181,27 @@ router.post('/replaceMediator/:id', api.isNat, api.isNotSpectator, async (req, r
     let v = await vetoesService.query({ _id: req.params.id }, defaultPopulate);
     let currentMediators = [];
     v.mediations.forEach(mediation => {
-        if(mediation.id != req.body.mediationId){
```

Comment:
you can still include the current mediation since you dont want to replace with the same user

Reply:
ye that's literally what it does after removing the ``if`` 🤔